### PR TITLE
Naive RNN acoustic model Debug & add HiFi-GAN vocoder & saving models Debug

### PR DESCRIPTION
**Pull request by PeterGuoRuc**

### CODE REVIEW

File: muskit/svs/muskit_model.py [hunk 1]
```diff
@@ -166,8 +183,33 @@ def forward(
             text_lengths=text_lengths,
             feats=feats,
             feats_lengths=feats_lengths,
+            flag_IsValid = flag_IsValid,
         )
 
+        # logging.info(f"pitch_lengths - DIO: {pitch_lengths}")
+        # logging.info(f"score_lengths - midi: {score_lengths}")
```
Comment (ftshijt) [inline muskit/svs/muskit_model.py:190]: in finalized pr, please remove the logging information

File: muskit/svs/muskit_model.py [hunk 2]
```diff
@@ -87,6 +87,7 @@ def forward(
         spembs: Optional[torch.Tensor] = None,
         sids: Optional[torch.Tensor] = None,
         lids: Optional[torch.Tensor] = None,
+        flag_IsValid = False,
```
Comment (ftshijt) [inline muskit/svs/muskit_model.py:90]: what's this for?

File: egs/TEMPLATE/svs1/svs.sh [hunk 3]
```diff
@@ -55,14 +55,15 @@ use_xvector=false    # Whether to use x-vector (Require Kaldi).
 feats_extract=fbank        # On-the-fly feature extractor.
 feats_normalize=global_mvn # On-the-fly feature normalizer.
 # Only used for feats_type != raw
-fs=16000          # Sampling rate.
```
Comment (ftshijt) [inline egs/TEMPLATE/svs1/svs.sh:58]: I don't think it's wise to directly change the svs.sh, please just change the specific egs

File: egs/TEMPLATE/svs1/svs.sh [hunk 4]
```diff
@@ -55,14 +55,15 @@ use_xvector=false    # Whether to use x-vector (Require Kaldi).
...
+fs=24000          # Sampling rate.
 fmin=80           # Minimum frequency of Mel basis.
-fmax=12000        # Maximum frequency of Mel basis.
+fmax=7600        # Maximum frequency of Mel basis.
 n_mels=80         # The number of mel basis.
 n_fft=1024        # The number of fft points.
 n_shift=256       # The number of shift points.
 win_length=null   # Window length.
 score_feats_extract=frame_score_feats # The type of music score feats (frame_score_feats or syllable_score_feats)
+pitch_extract=dio
```
Comment (ftshijt) [inline egs/TEMPLATE/svs1/svs.sh:66]: Do not set it as default

File: muskit/svs/muskit_model.py (hunk 1 above)
Comment (PeterGuoRuc) [reply muskit/svs/muskit_model.py:190]: ok！

File: muskit/svs/muskit_model.py (hunk 2 above)
Comment (PeterGuoRuc) [reply muskit/svs/muskit_model.py:90]: This is for generating intermediate results during the validation stage.
e.g. plotting Mel-spec and generating .wav from acoustic model & G.T.

File: egs/TEMPLATE/svs1/svs.sh (hunk 4 above)
Comment (PeterGuoRuc) [reply egs/TEMPLATE/svs1/svs.sh:66]: ok!

**Comment by ftshijt** [comment]
Can you try to fix the issues ASAP? It seems this is very important for others to continue working on

**Comment by ftshijt** [comment]
thanks！

**Pull request merged**